kodilla-patterns/src/main/java/com/kodilla/patterns/builder/bigmac/Bigmac.java

```diff
@@ -0,0 +1,162 @@
+package com.kodilla.patterns.builder.bigmac;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class Bigmac {
+
+    private final String bun;
+    private final int burgers;
+    private final String sauce;
+    private final List<String> ingredients;
+
+    public static class BigmacBuldier {
+
+        private String bun;
+        private int burgers;
+        private String sauce;
+        private List<String> ingredients = new ArrayList<>();
+
+        public BigmacBuldier bun(int chosen) {
+            switch (chosen) {
```

Comment:
default dajemy na koncu. a w ogole to nie wiem, czemu znow pokomplikowales ... po co te switche tutaj?